treescript/src/treescript/mercurial.py

```diff
@@ -35,15 +34,7 @@ def build_hg_command(config, *args):
     hg = config["hg"]
     if not isinstance(hg, (list, tuple)):
         hg = [hg]
-    robustcheckout_args = [
-        "--config",
-        "extensions.robustcheckout={}".format(ROBUSTCHECKOUT_PATH),
-        "--config",
-        "extensions.purge=",
-        "--config",
-        "extensions.strip=",
-    ]
-    return hg + [*robustcheckout_args, *args]
+    return hg + [*args]
```

Comment:
nit: Can just do `hg + args`

Reply:
At least the tests pass a tuple for `args`, and you can't add a tuple to a list. The `[*args]` syntax turns the argument into a list.